CONTRIBUTING.md

```diff
@@ -2,7 +2,8 @@ TigerOS
 ====================
 
 ## Contributing
-Please fork this repository and submit pull requests against the devel branch. When you do work in your fork, create a new branch for that work rther than wroking in a branch that has the same name as on in our repository. This helps to reduce occurences of the two repositories getting out of sync. For instructions on configuring your git environment to match with our workflow please see [here](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced) Pull requests will be reviewed as time permits.
+Please fork this repository and submit pull requests against the devel branch. When you do work in your fork, create a new branch for that work rather than working in the upstream branch. This helps prevent issues when merging your pull requests. For instructions on configuring your git environment to match with our workflow please see [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/) Pull requests will be reviewed as time permits.
```

Comment:
`Should make clear that the new branch is local and should not be pushed upstream to us.` - as in "create a new branch in your fork/local repository"
Also, "in the upstream branch" was an offhand suggestion for the idea, I think the wording still feels odd (albeit better). "rather than the branches from upstream/us" might be closer to what I envisioned.
These two together:
`When you do work in your fork, create a new branch in your repo rather than working in the branches coming from us.`
I ping @jflory7 as the pro writer to give his 2 cents.

Also, I still think that `putting an "additional reading" list below` would be helpful since we have multiple articles that are "couched" an then have the work "here" as the link, which makes a lot less sense than a bulleted list with the entire (official) article title being the text for the link